admin/app/utils/create-tree-data.js

```diff
@@ -0,0 +1,100 @@
+const RESULT_STATUSES = ['FAILED', 'STARTED', 'SUCCEEDED', 'SKIPPED'];
+
+import { fromUint8Array } from 'js-base64';
+import { deflate } from 'pako';
+
+const formatJSON = (data) => JSON.stringify(data, undefined, 2);
+const serialize = (state) => {
+  const data = new TextEncoder().encode(state);
+  const compressed = deflate(data, { level: 9 }); // zlib level 9
+  return fromUint8Array(compressed, true); // url safe base64 encoding
+};
+
+function createTreeFromData(data) {
+  console.log(data);
+  const relations = data.flatMap((pathWithNumber) =>
+    createRelationsFromPath(pathWithNumber.fullPath).map((path) => {
+      return { ...path, number: pathWithNumber.nombre };
+    }),
+  );
+  const uniqueRelations = [];
+
+  relations.forEach((relation) => {
+    const uniqueRelation = uniqueRelations.filter(
+      (uniqueRelations) => relation.from === relation.from && uniqueRelations.to === relation.to,
```

Comment:
`(uniqueRelation) => relation.from === relation.from && uniqueRelation.to === relation.to,`